orc8r/cloud/go/tools/combine_swagger/main.go

```diff
@@ -48,7 +49,11 @@ func main() {
 	}
 
 	fmt.Printf("Combining specs together...\n\n")
-	outSpec, warnings := spec.Combine(commonCfg, cfgs)
+	outSpec, err, warnings := swagger.Combine(commonCfg, cfgs)
+	if err != nil {
+		glog.Fatal(err)
+	}
+
```

Comment:
Remove newline -- logging the warnings is part of the same "paragraph", if you want to think of it that way